Gemfile

```diff
@@ -32,8 +32,6 @@ group :test do
   gem "test-theme", :path => File.expand_path("test/fixtures/test-theme", __dir__)
   gem "test-theme-skinny", :path => File.expand_path("test/fixtures/test-theme-skinny", __dir__)
   gem "test-theme-symlink", :path => File.expand_path("test/fixtures/test-theme-symlink", __dir__)
-  gem "tzinfo", ">= 1", "< 3"
```

Comment:
I moved tzinfo into the test group because the new TestWinTz tests run on all platforms. The tests are still passing on non-Windows because tzinfo is being brought in through another dependency.

Reply:
The `WinTZ` utility module is only used in Windows:
https://github.com/jekyll/jekyll/blob/f51ccbfad3e2df46a714a3c50b80f2556529c774/lib/jekyll.rb#L133-L139

So as of now, testing the module on non-Windows platforms is redundant.
That said, I was hoping to use the `WinTZ` in all platforms in a separate PR, after merging this.